### ISSUE TITLE
Equalizer Blocks Audio with Peak or Shelf Frequency Connected to Random LFO Controller

### ISSUE THREAD

**Issue by BiRD4** [LMMS/lmms#5099]
If the frequency control on an equalizer's peak or shelf (filters 2-7) is connected to an LFO controller set to random, the gain of the peak or shelf is not set to 0, and audio passes through it, the EQ will block the entire audio signal. Even after the controller is disconnected, this continues. This doesn't happen with the highpass and lowpass filters, and no other EQ controls seem to have this problem.

Using Windows 10 Home and LMMS 1.2.0 Stable

**Comment by BaraMGB** [comment]
Is this related to the missing oversampling?
https://github.com/LMMS/lmms/pull/4881

**Comment by BiRD4** [comment]
No. This is an issue with the current stable branch. I should also mention that the colors in the EQ visualizer still show the input, but it's just not giving any output.
![image](https://user-images.githubusercontent.com/20910668/62006048-938faf00-b109-11e9-89d1-52fea231b21c.png)

**Comment by PhysSong** [comment]
It sounds like #4127 which is fixed in the master branch by #4260(and a regression by #4396). We might backport those fixes if needed.
